src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTestTransformer.java

```diff
@@ -139,12 +146,15 @@ private SmokeTestClassView createSmokeTestClassView(SurfaceTransformerContext co
         context.asFlattenedMethodContext(method, flatteningGroup);
 
     SmokeTestClassView.Builder testClass = SmokeTestClassView.newBuilder();
+    TestMethodView testMethodView = createSmokeTestMethodView(methodContext);
+
     testClass.apiSettingsClassName(namer.getApiSettingsClassName(service));
     testClass.apiClassName(namer.getApiWrapperClassName(service));
     testClass.name(name);
     testClass.outputPath(namer.getSourceFilePath(outputPath, name));
     testClass.templateFileName(SMOKE_TEST_TEMPLATE_FILE);
-    testClass.method(createSmokeTestMethodView(methodContext));
+    testClass.method(testMethodView);
+    testClass.requireProjectID(requireProjectId(testMethodView.initCode(), context.getNamer()));
```

Comment:
We should be consistent with the casing - I think per Java convention, we should have "Id" (not "ID")